apps/users-api/src/routes/refreshToken.ts

```diff
@@ -0,0 +1,32 @@
+import { Router } from 'express';
+import { z } from 'zod';
+import jwt from 'jsonwebtoken';
+import { parseRequest, generateAccessToken, generateRefreshToken, verifyRefreshToken } from '@competition-manager/utils';
+
+export const router = Router();
+
+const Cookies$ = z.object({
+    refreshToken: z.string()
+});
+
+router.get(
+    '/refresh-token',
+    parseRequest('cookies', Cookies$),
+    async (req, res) => {
+        const { refreshToken } = Cookies$.parse(req.cookies);
+        const validRefreshToken = verifyRefreshToken(refreshToken);
+        const { email } = jwt.decode(refreshToken) as { email: string };
```

Comment:
Je pense plus un truc comme ceci
```suggestion
        const tokenData = TokenData$.parse(jwt.decode(refreshToken));
```